search/intermediate_result.cpp

```diff
@@ -248,8 +248,12 @@ void FillDetails(FeatureType & ft, Result::Details & details)
     /// @todo We should check closed/open time for specific feature's timezone.
     time_t const now = time(nullptr);
     if (oh.IsValid() && !oh.IsUnknown(now))
+    {
       details.m_isOpenNow = oh.IsOpen(now) ? osm::Yes : osm::No;
-    // In else case value us osm::Unknown, it's set in preview's constructor.
+      // In else case value is osm::Unknown, it's set in preview's constructor.
+      details.m_minutesUntilOpen = (oh.GetNextTimeOpen(now) - now) / 60;
```

Comment:
@vng should it be here or in the final results sent to UI? Won't it slow down the search?

Reply:
The same as details.m_isOpenNow, no problem here.

Reply:
Talking about speed, we can combine calculation of m_isOpenNow, m_minutesUntilOpen, m_minutesUntilClosed into one osmoh::OpeningHours call, but I don't think that we will have any significant gain here.